python3/totoro_agent/brain/pickup_tracker.py

```diff
@@ -10,6 +10,60 @@ def __init__(self):
     def update(self, game_state):
         # TODO gaurav
         # powerup - stop picking up powerups as soon as  diameter reaches double map size
+        # Couldnt do the above as we didnt have picking up functionality, and I didn't get 
+        # Why we needed to do it if it reached double map size. Will do tomorrow. 
         # ammo
 
-        pass
+        # Making empty lists for ammo and powerups. Will modify tick by tick.
+        ammo_list = []
+        powerup_list = []
+
+        # For loops for adding stuff to ammo_list and powerup_list 
+        for a in game_state['entities']:
+        	if a['type']=="a":
+        		x = a['x']
+        		y = a['y']
+        		ammo_list.append((x,y))
+
+        for b in game_state['entities']:
+        	if b['type']=="bp":
```

Comment:
Can just do this check into the first for-loop so we only go through entities once. No need to separate them

Reply:
Do something like
for i in game_state[..]:
if i['type']== 'a':
...
if i['type'] == 'b':